diff -up arm-trusted-firmware-2.12.0/tools/fiptool/Makefile.omv~ arm-trusted-firmware-2.12.0/tools/fiptool/Makefile
--- arm-trusted-firmware-2.12.0/tools/fiptool/Makefile.omv~	2025-01-03 20:10:55.600495684 +0100
+++ arm-trusted-firmware-2.12.0/tools/fiptool/Makefile	2025-01-03 20:11:26.756880729 +0100
@@ -47,7 +47,7 @@ DEFINES += -DUSING_OPENSSL3=$(USING_OPEN
 # directory. However, for a local build of OpenSSL, the built binaries are
 # located under the main project directory (i.e.: ${OPENSSL_DIR}, not
 # ${OPENSSL_DIR}/lib/).
-LDOPTS := -L${OPENSSL_DIR}/lib -L${OPENSSL_DIR} -lcrypto
+LDOPTS := -lcrypto
 INCLUDE_PATHS += -I${OPENSSL_DIR}/include
 endif # STATIC
 
